requirements.txt

```diff
@@ -18,7 +18,7 @@ python-dateutil==2.8.2
 python-hglib==2.6.2
 ratelimit==2.2.1
 requests==2.26.0
-rs_parsepatch==0.3.5
+rs_parsepatch==0.3.6
```

Comment:
Unrelated change (maybe leftover from merge?)

Reply:
Right, thanks, I've reverted the change